python/ccf/clients.py

```diff
@@ -76,27 +84,28 @@ def makefile(self, *args, **kwargs):
 
 
 class Response:
-    def __init__(self, status, body, seqno, view, global_commit, headers):
-        self.status = status
+    """
+    Response to request sent via :py:class:`ccf.clients.CCFClient`
+    """
+
+    def __init__(self, status_code, body, seqno, view, global_commit, headers):
```

Comment:
Response would probably make a nice dataclass

Reply:
Addressed in #1429